### ISSUE TITLE
Guidelines to choose any npm package into my project.

### ISSUE THREAD

**Issue by NaviMarella** [kentcdodds/ama#438]
Hi Kent,

Can you please explain what are the different guidelines, criteria we should follow in choosing an npm package into our project.

**Comment by hedgerh** [comment]
You forgot to add a description

**Comment by hedgerh** [comment]
Marked invalid.  Will close if no description is provided in a few hours.

**Comment by kentcdodds** [comment]
Actually, I think I have a good idea what the asker is looking for. Please leave this one open.

**Comment by hedgerh** [comment]
No problem.  Looks like they added their question.  It was originally blank with just a title.

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
